plugin-directorymenu/directorymenu.cpp

```diff
@@ -87,7 +86,6 @@ void DirectoryMenu::buildMenu(const QString& path)
     if(mMenu)
     {
         delete mMenu;
-        mMenu = 0;
```

Comment:
This removal is also wrong and will cause SEGFAULTS IMO.

Reply:
`mMenu = nullptr;` or `mMenu = 0;`?

Reply:
Oh.. I forgot to look further... the `mMenu` is assigned right after....

Reply:
The same here... just `delete mMenu;`